libcontainer/configs/cgroup_linux.go

```diff
@@ -127,6 +127,6 @@ type Resources struct {
 	// CpuWeight sets a proportional bandwidth limit.
 	CpuWeight uint64 `json:"cpu_weight"`
 
-	// CpuMax sets she maximum bandwidth limit (format: max period).
-	CpuMax string `json:"cpu_max"`
+	// CpuMax is converted from CpuQuota and CpuPeriod.
+	// Specifying CpuMax directly is not supported currently.
```

Comment:
I think we should just remove it, since it is not in spec (as it was done in https://github.com/opencontainers/runc/pull/2428/commits/165e76c9253ff1860aeb96a26dcd6ae3813dcc1b)